meetings/2022-12-05.md

```diff
@@ -0,0 +1,91 @@
+# 2022-12-05 Solid Interoperability
```

Comment:
from [README.md](https://github.com/solid/data-interoperability-panel/blob/main/README.md#meetings)
> Panel meetings are held most Tuesdays at 10:00AM Eastern / 4:00PM Central European. Check-in on [gitter](https://gitter.im/solid/data-interoperability-panel) for weekly agenda.

It's too bad today isn't a Tuesday, and no "agenda" was committed until it was these "minutes".

Today's [Gitter log](https://gitter.im/solid/data-interoperability-panel) suggests the [README.md](https://github.com/solid/data-interoperability-panel/blob/main/README.md#meetings) should be updated to reflect "every other Monday" instead of "most Tuesdays".

Also, the [Solid CG calendar](https://www.w3.org/groups/cg/solid/calendar) DESPERATELY needs updates to include all regularly scheduled meetings (for whole CG, specific panels, just editors, etc.). Further, it _should_ be possible to add special meetings as soon as they're scheduled.